public-resources/columns.csv

```diff
@@ -116,12 +229,27 @@ Enrollment,68,DateUpdated,datetime,0,0
 Enrollment,69,UserID,character,0,0
 Enrollment,70,DateDeleted,datetime,0,1
 Enrollment,71,ExportID,character,0,0
+Event,1,EventID,character,1,0
+Event,2,EnrollmentID,character,1,0
+Event,3,PersonalID,character,1,0
+Event,4,EventDate,date,1,0
+Event,5,Event,numeric,0,0
```

Comment:
This feels like maybe it should be HP? We use Event when calculating oustanding referrals (id=100). If Event is Null, we risk not flagging cases. But maybe that's okay; We'll flag if needed once they fill in Event, as long as they notice and take care of null errors.

Reply:
I agree, it's an important data element and it should be shifted back to High Priority soon. For now we have to shift it to Error per JJ.